pretrained_bert/dataset_reader.py

```diff
@@ -140,6 +140,7 @@ def is_whitespace(char):
                 break
             start_offset += min(length, self._document_stride)
 
+        #
```

Comment:
Empty comment?

Reply:
oh, I forgot to delete the empty comment and I have fixed.